openbb_terminal/stocks/stocks_controller.py

```diff
@@ -490,9 +490,11 @@ def call_news(self, other_args: List[str]):
         )
         if ns_parser:
             if self.ticker:
-                if ns_parser.source == "NewsApi":
+                try:
                     d_stock = yf.Ticker(self.ticker).info
-
+                except TypeError:
+                    d_stock = dict()
```

Comment:
Wont this break the below line because it wants d_stock["shortName"].  IF there is a value error, you will get a keyerror below

Reply:
No, due to this row: https://github.com/OpenBB-finance/OpenBBTerminal/blob/f64d1b812608d91e0a17385c9ffea766c853274e/openbb_terminal/stocks/stocks_controller.py#L511, it checks if "shortName" is in the dictionary.

Reply:
I see. Silly formatting putting the if on the next line.